src/wl-clipboard.h

```diff
@@ -0,0 +1,12 @@
+#ifndef WL_CLIPBOARD_H
+#define WL_CLIPBOARD_H
+
+#include <QString>
+#include <optional>
+
+namespace wl_clipboard {
+    bool setClipboard(const QString& text);
+    std::optional<QString> getClipboard();
```

Comment:
```suggestion
    bool set_clipboard(const QString& text);
    std::optional<QString> get_clipboard();
```

Sorry for being nitpicky. This project doesn't use CamelCase, but snake_case.

Please rename files to snake_case as well: `wl_clipboard.hpp` and `wl_clipboard.cpp`. Use hpp file extension for c++ header.

Reply:
Not a problem